__tests__/app.test.js

```diff
@@ -158,8 +161,54 @@ describe("GET all article comments", () => {
       .get("/api/articles/123456789/comments")
       .expect(404)
       .then(({ body }) => {
+        expect(body.msg).toBe("Not found");
+      });
+  });
+});
 
+describe('POST /api/articles/:article_id/comments/', () => {
+  test('should add a comment to an article ', () => {
+    return request(app)
+      .post("/api/articles/2/comments/")
+      .send({
+        body: "Laptops are useful!",
+        author: "rogersop",
+        article_id: 2
+      })
+      .expect(200)
+      .then(({ body}) => {
+        expect(body).toMatchObject({
+          comment_id: 19,
+          body: 'Laptops are useful!',
+          article_id: 2,
+          author: 'rogersop'
+        })
+      })
+  });
+  test("404:error when an incorrect article ID is entered e.g.'123456789' ", () => {
+    return request(app)
+      .post("/api/articles/123456789/comments")
+      .send({
+        body: "I like pears!",
+        author: "rogersop",
+        article_id: 123456789
+      })
+      .expect(404)
+      .then(({ body }) => {
         expect(body.msg).toBe("Not found");
       });
   });
+
+  test('should return a bad request error if no comment is sent ', () => {
+    return request(app)
+      .post("/api/articles/2/comments")
+      .send({
+        author: "rogersop",
+        article_id: 2
```

Comment:
I would have the article_ id in a separate test if you want to check putting it in the object doesnt effect the post